src/ParallelAlgorithms/Interpolation/Protocols/PostInterpolationCallback.hpp

```diff
@@ -33,14 +31,14 @@ struct has_signature_1<
     T, std::void_t<decltype(T::apply(
            std::declval<const db::DataBox<DummyTag>&>(),
            std::declval<const Parallel::GlobalCache<DummyMetavariables>&>(),
-           std::declval<const ::Tags::Time&>()))>> : std::true_type {
+           std::declval<const TimeStepId&>()))>> : std::true_type {
```

Comment:
Don't use `TimeStepId` here.  It looks reasonable but is almost always wrong, so don't use it where someone might look for an example.

Reply:
Made it `double` here and in ComputeTargetPoints